python/lsst/ts/linearstage/controllers/igus_dryve/igus_dryve.py

```diff
@@ -1087,3 +1084,49 @@ async def send_telegram(
 
                 if return_response:
                     return data
+
+    @classmethod
+    def get_config_schema(cls):
+        return yaml.safe_load(
+            """
+            $schema: http://json-schema.org/draft-07/schema#
+            $id: https://github.com/lsst-ts/ts_LinearStage/master/python/lsst/ts/linearstage/config_schema.py
+            title: Igus v1
+            description: Sub-schema for Igus configuration files
+            type: object
+            properties:
+                igus_controller:
```

Comment:
This property not needed. We already know this config is for the igus controller. I strongly suggest deleting it and moving the remaining entries up one level.